tests/PHPUnit/Unit/DateTest.php

```diff
@@ -485,6 +485,14 @@ public function getTestDataForIsTodayTest()
             'Pacific/Auckland',
             'Pacific/Apia',
             'Pacific/Kiritimati',
+            'UTF-1',
```

Comment:
Shouldn't these be `UTC` timezones not `UTF` timezones?

Reply:
Whoops! I thought I fixed that. It's fixed now. Thank you.